assets/syntaxes/Packages/Git Formats/Completions/Git Attributes - Encoding.sublime-completions

```diff
@@ -29,14 +29,14 @@
         {"trigger": "Windows 1256\tArabic", "contents": "windows-1256"},
         {"trigger": "Windows 1257\tBaltic", "contents": "windows-1257"},
         {"trigger": "Windows 1258\tVietnamese", "contents": "windows-1258"},
-        {"trigger": "big5\tChineese Traditional", "contents": "gb2312"},
+        {"trigger": "big5\tChinese Traditional", "contents": "gb2312"},
```

Comment:
These typos appear to be present in the original: https://github.com/sublimehq/Packages/blob/c7cd5a7dc3eb34b5746b2f87c5ca4967574c9e40/Git%20Formats/Completions/Git%20Attributes%20-%20Encoding.sublime-completions#L174 

I'll probably send a PR to them...